src/schema.py

```diff
@@ -141,12 +142,13 @@ def resolve_account_info(self, info, session_id=None):
         continue
       dt_utc = parser.parse(t['actualDate']).astimezone(pytz.timezone('UTC'))
```

Comment:
Are we using `dt_utc`? If not, why not `pytz.timezone('EST')` and just set `dt_est` from the start? Also, prefer more verbose variable names like `datetime_est`